playbooks/roles/openvpn/tasks/main.yml

```diff
@@ -54,6 +54,15 @@
   args:
     creates: "{{ openvpn_ca }}.crt"
 
+- name: Generate a random server common name
+  shell: grep -v -P "[\x80-\xFF]" /usr/share/dict/american-english-huge | sed -e "s/'//" | shuf -n 2 | xargs | sed -e 's/ /./g'
```

Comment:
Although most people only run the playbook once, it's best if tasks are idempotent and can safely run multiple times. Currently, this task will generate a different random common name on every single run and change the server configuration to match, which could cause problems for clients that have already been configured to expect a certain name.

One pattern that happens in several roles in Streisand is to place random output in a file, and then skip the random generation task if the file already exists. Then a second task registers the output from the first. You can see an example of how this pattern works in the OpenConnect role where a random password is created [here](https://github.com/jlund/streisand/blob/master/playbooks/roles/openconnect/tasks/main.yml#L109) and registered [here](https://github.com/jlund/streisand/blob/master/playbooks/roles/openconnect/tasks/main.yml#L121).

If we followed the same pattern for this, we can ensure that the server name remains constant no matter how many times the Streisand playbook is run against a server.